src/pages/RegionsBoard/RegionBoardDocuments.tsx

```diff
@@ -40,20 +40,26 @@ const RegionBoardDocuments = () => {
   const [userAccesses, setUserAccesses] = useState<{[key: string] : boolean}>({});
 
   const getUserAccesses = async () => {
+    setLoading(true);
+    debugger;
```

Comment:
remove it
App will hangs in production with this

Reply:
FYI
if you create PR, but it's still in progress, consider of using draft pull-requests
https://github.blog/2019-02-14-introducing-draft-pull-requests/

Reply:
done, forgot about this